stable/rabbitmq-ha/templates/statefulset.yaml

```diff
@@ -48,7 +48,7 @@ spec:
               containerPort: 5672
             - name: http
               protocol: TCP
-              containerPort: 15672
+              containerPort: {{ .Values.rabbitmqManagerPort }}
```

Comment:
Please add to values.yaml

Reply:
Also add default value to readme

Reply:
It already is in both values.yaml and readme, just was not used here.

Reply:
Thanks!